core/sdk/src/api/distributor/distributor_privacy.rs

```diff
@@ -300,6 +300,23 @@ impl FromStr for DataEventType {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
```

Comment:
Confirm if both Serialize and Deserialize trait is needed here.

Reply:
Not needed and removed them.